openedx/core/djangoapps/catalog/utils.py

```diff
@@ -140,8 +140,10 @@ def get_programs_with_type(include_hidden=True):
     if programs:
         program_types = {program_type['name']: program_type for program_type in get_program_types()}
         for program in programs:
-            if program['type'] not in program_types:
-                continue
+
+            if not waffle.switch_is_active('get-multitenant-programs'):
```

Comment:
@douglashall , we are looping through all the programs and skipping the program-types not in our allowed list of program-types. If we move the switch outside, all the programs will be skipped.